src/altinn-app-frontend/src/utils/formLayout.ts

```diff
@@ -106,7 +106,7 @@ export function getRepeatingGroups(formLayout: ILayout, formData: any) {
   });
 
   // filter away groups that should be rendered as child groups
-  const filteredGroups = formLayout.filter((group) => childGroups.indexOf(group.id) === -1);
+  const filteredGroups = groups.filter((group) => childGroups.indexOf(group.id) === -1);
```

Comment:
Just noticed this as well. Is it supposed to check all elements in the formlayout and only filter away children of groups? Or is it sufficient to start with only the groups before filtering. It is called filteredGroups after all 🤷